### ISSUE TITLE
Crystal drops are determined incorrectly

### ISSUE THREAD

**Issue by isotor** [DarkstarProject/darkstar#5847]
**_I have:_**

- [x] searched existing issues (http://github.com/darkstarproject/darkstar/issues/) to see if the issue I am posting has already been addressed or opened by another contributor
- [x] checked the commit log to see if my issue has been resolved since my server was last updated

**_Client Version_** (type `/ver` in game) **:** 30190328_2

**_Source Branch_** (master/stable) **:** master

**_Additional Information_** (Steps to reproduce/Expected behavior) **:** 

Issue discovered by Eden player roclm [here](https://github.com/EdenServer/community/issues/1299). The relevant extract:

*"As it stands right now, you cannot obtain crystals in regions in which the Beastmen are the Dominant influence. This is incorrect behavior, crystals should still be obtainable in regions in which the Beastmen are the Dominant influence as long as one of the three Nations controls the region."*

tl;dr: crystlal drops should be determined by region control as assigned during the weekly conquest tally, and not by current level of influence.

**Comment by isotor** [comment]
Relevant section: https://github.com/DarkstarProject/darkstar/blob/2924080116c08539ab245f2a8947f1d448237bdf/src/map/entities/mobentity.cpp#L840

Would replacing `GetInfluenceGraphics` with `GetRegionOwner` do the trick?
